tests/testthat/test-RandMat.R

```diff
@@ -0,0 +1,239 @@
+context("RandMat* tests")
+library(rerf)
+
+test_that("Testing RandMat binary option.", {
+  ## below is the output of RandMat with mat.options
+  ## from commit 73b896ff053537ee23d82b9debee054171b1c41b
+  ## with set.seed(317) and RcppZiggurat::zsetseed(14)
+  ## for comparison to the new version of RandMat*
+  #mat.options <- list(p = 5, d = 3, "binary", rho = 0.25, prob = 0.5)
+  rBinary <- structure(c(3, 2, 3, 2, 1, 2, 2, 3, 1, -1, -1, 1), .Dim = 4:3)
+
+  set.seed(317)
+  RcppZiggurat::zsetseed(14)
+  #optBinary <- list(p = 5, d = 3, "binary", rho = 0.25, prob = 0.5)
+  #a01 <- RandMat(optBinary)
+  optBinary <- list(p = 5, d = 3, rho = 0.25, prob = 0.5)
+  a01 <- do.call(RandMatBinary,optBinary)
+  expect_equal(a01, rBinary)
+})
+
+
+test_that("Testing RandMat continuous option.", {
+  ## below is the output of RandMat with mat.options
+  ## from commit 73b896ff053537ee23d82b9debee054171b1c41b
+  ## with set.seed(317) and RcppZiggurat::zsetseed(14)
+  ## for comparison to the new version of RandMat*
+  #mat.options <- list(p = 5, d = 3, "continuous", rho = 0.25, prob = 0.5)
+  rContinuous <- structure(c(3, 2, 3, 2, 1, 2, 2, 3, -0.42934944005662,
+                             -1.00656832180404, -0.346220301300449,
+                             -2.88694544455467), .Dim = 4:3)
+  set.seed(317)
+  RcppZiggurat::zsetseed(14)
+  #optContinuous <- list(p = 5, d = 3, "continuous", rho = 0.25, prob = 0.5)
+  #a02 <- RandMat(optContinuous)
+  optContinuous <- list(p = 5, d = 3, rho = 0.25)
```

Comment:
`rho` is not a parameter for randmatcontinuous (get an error when I try to run testthat)

Reply:
fixed